### PR TITLE
Changelog component restyle

### PR DESCRIPTION
**Pull request by O-Susanoo**

### CODE REVIEW

File: src/components/changelog/Changelog.tsx [hunk 1]
```diff
@@ -59,52 +61,70 @@ type Props = RouteComponentProps & StateProps & StyleProps & DispatchProps
...
+                <Table.Cell>
+                  <List.Item>
+                    <List.Content>
+                      <List.Header style={{ paddingTop: "10px" }}>
+                        <Image avatar src={entry.user.avatarUrl} />
+                        {entry.user.login}
+                      </List.Header>
+                      <List.Description style={{ marginTop: "20px", marginBottom: "20px" }}>
+                        <div className="ui content header">{entry.title}</div>
+                        <div style={{ color: '#00000066', marginTop: '-10px', fontSize: '1em' }}>Published {new Date(entry.closedAt).toUTCString()}</div>
```
Comment (vdiezel) [inline src/components/changelog/Changelog.tsx]: We should probably use `jss` here! What do you think? :thinking:

File: test/components/changelog/Changelog.test.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        const callRemoveNotificationsByTopicMock = jest.fn()
+        nexus.user.getChangelog = getChangelogMock
+        thunks.callRemoveNotificationsByTopic = callRemoveNotificationsByTopicMock
+        const { getByText } = renderWithStore()
+        await wait(() => {
+            const result = getByText('vdiezel')
+            expect(result).not.toBeNull()
+        }, { timeout: 100 })
+    })
+})
```
Comment (vdiezel) [inline test/components/changelog/Changelog.test.js]: love this. Even with delay!

Comment (O-Susanoo) [reply test/components/changelog/Changelog.test.js]: Yup i kind of start liking testing-library/react

**Comment by O-Susanoo** [comment]
> Looks go to go! Implement `jss` at your own judgement :)

I like to use big guns only when it's necessary

**Pull request merged**